komodo/cli.py

```diff
@@ -340,13 +362,16 @@ def cli_main():
         "shell script templates.",
     )
 
-    args = parser.parse_args()
+    args = parser.parse_args(args)
 
-    if args.workspace and not Path(args.workspace).exists():
-        Path(args.workspace).mkdir()
+    if args.pyver is not None:
+        message = (
+            "\n⚠️  The --pyver option is deprecated and will be removed in a "
+            "future version of komodo. It is not used by komodo.\n"
+        )
+        print(message, file=sys.stderr)
```

Comment:
You could use a `FutureWarning` for this; https://docs.python.org/3/library/exceptions.html#FutureWarning

Reply:
Ok, that is much better :raised_hands: I assumed it would have the same problem as `DeprecationWarning` but no, it shows up just fine. I guess it must be [the default warning filters](https://docs.python.org/3/library/warnings.html#default-warning-filter). TIL.